services/s3/src/it/java/software/amazon/awssdk/services/s3/CopyObjectIntegrationTest.java

```diff
@@ -87,9 +87,10 @@ public static void initializeTestData() throws Exception {
     @Test
     public void copyObject_CopiesObjectToNewKey() throws Exception {
         s3.copyObject(CopyObjectRequest.builder()
-                                       .copySource(BUCKET_NAME + "/" + SOURCE_KEY)
-                                       .bucket(BUCKET_NAME)
-                                       .key(DESTINATION_KEY)
+                                       .sourceBucket(BUCKET_NAME)
```

Comment:
Why did we remove the ` .copySource(BUCKET_NAME + "/" + SOURCE_KEY)`  , can we keep the test case and assert behavior when user  calls copyObject with ` .copySource(BUCKET_NAME + "/" + SOURCE_KEY)`

Reply:
The test as it used to be was just testing the service. It's probably not a test we should have. Using sourceBucket and sourceKey is at least testing our creation of the copySource field.